File: dehaze-tool/dehaze-sdk-android/src/main/java/com/pei/dehaze/sdk/model/dict/DictTypeForm.java
package com.pei.dehaze.sdk.model.dict;

import lombok.Data;

/**
 * 字典表单模型类
 */
@Data
public class DictTypeForm {
    /**
     * 字典类型ID
     */
    private Integer id;
    
    /**
     * 类型名称
     */
    private String name;
    
    /**
     * 类型编码
     */
    private String code;
    
    /**
     * 类型状态：1:启用;0:禁用
     */
    private int status;
    
    /**
     * 备注
     */
    private String remark;
}

File: dehaze-go/model/bo/dict_type_form.go
package bo

// DictTypeFormBO 字典类型表单对象
type DictTypeFormBO struct {
	// 字典类型ID
	ID *int64 `json:"id"`
	// 类型名称
	Name string `json:"name"`
	// 类型编码
	Code string `json:"code"`
	// 类型状态(1:启用;0:禁用)
	Status int8 `json:"status"`
	// 备注
	Remark string `json:"remark"`
}
